proto/batch_plan.proto

```diff
@@ -72,6 +72,7 @@ message InsertNode {
   repeated uint32 column_idxs = 2;
   // Id of the materialized view which is used to determine which compute node to execute the dml fragment.
   uint32 associated_mview_id = 3;
+  bool returning = 4;
```

Comment:
We can add support for `DELETE` and `UPDATE` after the current implementation is polished well. 🤤

Reply:
supported!